src/test/java/seedu/guest/model/util/SampleDataUtilTest.java

```diff
@@ -57,27 +63,32 @@ public void getSampleGuestBook() {
         exceptedGb.addGuest(new Guest(new Name("James Oliver"), new Phone("98345344"),
                 new Email("jamesoliver@example.com"), new Room("04-73"),
                 new DateRange("15/09/22 - 19/09/22"),
-                new NumberOfGuests("2"), new IsRoomClean("no"), new Bill()));
+                new NumberOfGuests("2"), new IsRoomClean("no"), new Bill(), new Request()));
         exceptedGb.addGuest(new Guest(new Name("Kayla Smith"), new Phone("91823471"),
                 new Email("kaylasmith@example.com"), new Room("05-25"),
                 new DateRange("22/01/23 - 25/01/23"),
-                new NumberOfGuests("3"), new IsRoomClean("no"), new Bill("10")));
+                new NumberOfGuests("3"), new IsRoomClean("no"), new Bill("10"),
+                new Request("I want to add breakfast service")));
         exceptedGb.addGuest(new Guest(new Name("Charlie Lim"), new Phone("88237126"),
                 new Email("charlie@example.com"), new Room("05-63"),
                 new DateRange("21/02/22 - 22/03/22"),
-                new NumberOfGuests("2"), new IsRoomClean("yes"), new Bill("19.99")));
+                new NumberOfGuests("2"), new IsRoomClean("yes"), new Bill("19.99"),
+                new Request("I want to have dinner in my room")));
         exceptedGb.addGuest(new Guest(new Name("Ryan Wong"), new Phone("82831111"),
                 new Email("wongryan@example.com"), new Room("06-23"),
                 new DateRange("28/04/23 - 29/04/23"),
-                new NumberOfGuests("2"), new IsRoomClean("no"), new Bill("50")));
+                new NumberOfGuests("2"), new IsRoomClean("no"), new Bill("50"),
+                new Request("My bed is not comfortable")));
         exceptedGb.addGuest(new Guest(new Name("Ibrahim Irfan"), new Phone("84912293"),
                 new Email("irfan@example.com"), new Room("06-33"),
                 new DateRange("23/05/23 - 03/06/23"),
-                new NumberOfGuests("1"), new IsRoomClean("yes"), new Bill("999.99")));
+                new NumberOfGuests("1"), new IsRoomClean("yes"), new Bill("999.99"),
+                new Request("Could you help me to do the team project?")));
         exceptedGb.addGuest(new Guest(new Name("Dave Balakrishnan"), new Phone("92821374"),
                 new Email("daveb@example.com"), new Room("07-55"),
                 new DateRange("30/12/22 - 02/01/23"),
-                new NumberOfGuests("3"), new IsRoomClean("no"), new Bill("1000.01")));
+                new NumberOfGuests("3"), new IsRoomClean("no"), new Bill("1000.01"),
+                new Request("Could your officer help me complete the assignments, it is too hard.")));
```

Comment:
Should we make this more related to the hotel?

Reply:
changed